### PR TITLE
"resources" on single resource declarations is optional

### PR DESCRIPTION
**Pull request by soyuka**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | ∅
| License       | MIT
| Doc PR        | ∅

https://github.com/api-platform/core/pull/568#issuecomment-225510491

@teohhanhui

### CODE REVIEW

File: src/Metadata/Resource/Factory/XmlResourceMetadataFactory.php [hunk 1]
```diff
@@ -127,6 +125,31 @@ public function create(string $resourceClass) : ResourceMetadata
     }
 
     /**
+     * Creates a DOMDocument based on `resource` tags of a file-loaded ml document.
```
Comment (teohhanhui) [inline src/Metadata/Resource/Factory/XmlResourceMetadataFactory.php]: xml?

File: src/Metadata/Resource/Factory/XmlResourceMetadataFactory.php [hunk 2]
```diff
@@ -93,6 +89,8 @@ public function create(string $resourceClass) : ResourceMetadata
             return $this->handleNotFound($parentResourceMetadata, $resourceClass);
         }
 
+        $xpath = new \DOMXpath($resources);
```
Comment (teohhanhui) [inline src/Metadata/Resource/Factory/XmlResourceMetadataFactory.php]: Unused.

**Comment by teohhanhui** [comment]
ping @api-platform/core-team

**Comment by dunglas** [comment]
👍

**Comment by dunglas** [comment]
Thank you @soyuka

**Pull request merged**